AutoGear.toc

```diff
@@ -8,4 +8,4 @@
 ## Notes: Automatically chooses loot and equips gear
 
 AutoGear.lua
-## SavedVariables: AutoGearDB
\ No newline at end of file
+## SavedVariables: AutoGearDB

```

Comment:
Oh, yikes.  Good catch here.  I hadn't noticed your previous pull request removed this.  I only barely skimmed it.  Thanks for re-adding this.  Too bad testers would have had their AutoGearDB wiped.  :(